docs/index.md

```diff
@@ -1 +1,95 @@
+---
+layout: index
+---
 
+## Overview
+
+Marquez is an open source **metadata service** for the **collection**, **aggregation**, and **visualization** of a data ecosystem's metadata. It maintains the [provenance](https://en.wikipedia.org/wiki/Provenance#Data_provenance) of how datasets are consumed and produced, provides global visibility into job runtime and frequency of dataset access, centralization of dataset lifecycle management, and much more. Marquez<sup>[1](#1)</sup> was released and open sourced by [The We Company](https://www.we.co).
+
+## Why Marquez?
+
+Marquez enables highly flexible data lineage queries across _all datasets_, while reliably and efficiently associating (_upstream_, _downstream_) dependencies between jobs and the datasets they produce and consume.
+
+### Features
+
+* Centralized **metadata management** powering:
+  * [Data lineage](https://en.wikipedia.org/wiki/Data_lineage)
+  * [Data governance](https://en.wikipedia.org/wiki/Data_governance)
+  * Data health
+  * Data discovery **+** exploration
+* Precise and highly dimensional [data model](#data-model)
+  * Jobs
+  * Datasets 
+* Easily collect metadata via an opinionated [Metadata API](./openapi.html)
+* **Datasets** as first-class values
+* **Enforcement** of _job_ and _dataset_ ownership
+* Simple operation and design with minimal dependencies
+* RESTful API enabling sophisticated integrations with other systems:
+  * [Airflow](https://airflow.apache.org)
+  * [Amundsen](https://github.com/lyft/amundsenfrontendlibrary)
+  * [Dagster](https://github.com/dagster-io/dagster)
+* Designed to promote a **healthy** data ecosystem where teams within an organization can seamlessly _share_ and _safely_ depend on one another's datasets with confidence
+
+## Why manage and utilize metadata?
+
+<figure align="center">
+  <img src="./assets/images/ecosystem.png">
+</figure>
+
+## Design
+
+Marquez is a modular system and has been designed as a highly scalable, highly extensible platform-agnostic solution for [metadata management](https://en.wikipedia.org/wiki/Metadata_management). It consists of the following system components:
```

Comment:
Metadata management was brought up above in the Features list, maybe link it up there instead of here?

Reply:
Agreed! I'll link the definition when the _metadata management_ concept is first introduced.